kernel/panic.c

```diff
@@ -210,7 +210,7 @@ void panic(const char *fmt, ...)
 	if (len && buf[len - 1] == '\n')
 		buf[len - 1] = '\0';
 
-	pr_emerg("Kernel panic - not syncing: %s\n", buf);
+	pr_emerg("Bruh moment - not syncing: %s\n", buf);
```

Comment:
bruh...

Reply:
bruh

Reply:
certified bruh moment

Reply:
bruh

Reply:
bruh

Reply:
b r u h

Reply:
b r u h
r
u
h

Reply:
Okay, this is epic.

Reply:
Bruhhhhhh

Reply:
bruh

Reply:
BRUH

Reply:
Bruh

Reply:
brUH

Reply:
Bruh

Reply:
bruh

Reply:
bruh...

Reply:
bruh

Reply:
god damn it, leave the best message kernel alone , tnxbai

Reply:
bruh

Reply:
![image](https://user-images.githubusercontent.com/29672562/59164047-d639ff80-8b3a-11e9-8105-dbde0356dc2b.png)

Reply:
> ![image](https://user-images.githubusercontent.com/29672562/59164047-d639ff80-8b3a-11e9-8105-dbde0356dc2b.png)

me and the boys when we trigger a kernel panic

Reply:
Bruh

Reply:
bruh

Reply:
bruh

Reply:
Now that's an epic bruh moment

Reply:
bruh!

Reply:
bruh!

Reply:
me and the bruhs

Reply:
bruh moment: systemctl crashed

Reply:
bruh bruh bruh bruh bruh

Reply:
bruh

Reply:
bruh

Reply:
bruh

Reply:
bruh

Reply:
"The kernel bruhs itself in confusion!"

Reply:
bruh

Reply:
merge this pull request ktnxbye

Reply:
bruh

Reply:
bruh

Reply:
bruh

Reply:
ok this is epic

Reply:
Bruh

Reply:
What a great moment !

Reply:
Bruh

Reply:
Such an improvement, love it :joy:

Reply:
obligatory BRUH

Reply:
Bruh

Reply:
Bruh momentos numero dos

Reply:
### bruh

Reply:
I love this ❤️ 
+1 from me!

Ps: I'm an happy Arch user.

Reply:
I hope linus approves this :+1:

Reply:
# bruh-b-wan-kenobi

Reply:
```css
bruh
```

Reply:
![bruh](https://user-images.githubusercontent.com/8678891/65634447-e4773300-dfe6-11e9-9f0c-bdb9506a7f0c.gif)

Reply:
Bruh

Reply:
bruh

Reply:
bruh

Reply:
🅱🆁🆄🅷

Reply:
> 
> 
> 🅱🆁🆄🅷

# 𝕓𝕣𝕦𝕙

Reply:
b̾r̾u̾h̾

Reply:
𝖇̶̬͙̭̟͎ͨͯͭ͆͋͘͘͝͝𝖗̨̦̘͇̘̥ͬ̔͗͐͋̀̀̀͢𝖚̡̨̯͇͎̥̫̈́̔̂̆͌̀͟͢𝖍̷̵̷̛̰̱͈̱͚ͥ̄̈́̍̎͘

Reply:
hurb